app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from .cafe import Cafe
+from .errors import VaccineError, NotWearingMaskError
```

Comment:
According to the checklist, you should use absolute imports instead of relative imports. Change `from .cafe import Cafe` and `from .errors import VaccineError, NotWearingMaskError` to `from app.cafe import Cafe` and `from app.errors import VaccineError, NotWearingMaskError`.